File: php/route.php
<?php
    class Route {
        public $id;
        public $start;
        public $end;

        public function __construct ($id, $start, $end) {
            $this->id = $id;
            $this->start = $start;
            $this->end = $end;
        }
    }
?>

File: java/Route.java
import java.util.ArrayList;

public class Route {
    Integer id;
    ArrayList<Double> start;
    ArrayList<Double> end;

    public Route (Integer id, ArrayList<Double> start, ArrayList<Double> end) {
        this.id = id;
        this.start = start;
        this.end = end;
    }
}